Dockerfile.windows

```diff
@@ -208,6 +211,40 @@ RUN `
     } `
   } `
   `
+  Function Build-GoTestSum() { `
+    $GotestsumPath=Join-Path -Path $Env:GOPATH -ChildPath "src\gotest.tools\gotestsum"; `
+    $GotestsumBinPath=Join-Path -Path $GotestsumPath -ChildPath "gotestsum.exe"; `
```

Comment:
Looks like this one isn't used anymore as well

Reply:
Removed